pdns/rfc2136handler.cc

```diff
@@ -869,9 +869,15 @@ int PacketHandler::processUpdate(DNSPacket& p) {
       if (rr->d_place == DNSResourceRecord::AUTHORITY) {
         /* see if it's permitted by policy */
         if (this->d_update_policy_lua != nullptr) {
+          const bool rfc2136_strict_mode = ::arg().mustDo("lua-dnsupdate-policy-script-strict-mode");
```

Comment:
I think this could even be `static`, to avoid the call every time.

Reply:
https://github.com/PowerDNS/pdns/pull/11285/commits/2b6b2f0358617a7f089f3f50fa4afb2e435a6099 changed here to a `static bool`